nyuntam_docs/docs/support.md

```diff
@@ -0,0 +1,67 @@
+
+| Model              | CPU PTQ - Torch | CPU PTQ - OpenVino | CPU PTQ - ONNX | GPU PTQ - TensorRT | CPU QAT - Torch | Knowledge Distillation | Structured Pruning | CPU QAT - OpenVino |
+|------------------------------|-----------------|--------------------|----------------|--------------------|-----------------|------------------------|--------------------|--------------------|
+| Resnet (timm)                | ✓               | ✓                  | ✓              | ✓                  | ✓               | ✓                      | ✓                  | ✓                  |
+| Convnextv2 (huggingface)     | -               | ✓                  | ✓              | ✓                  | -               | ✓                      | ✓                  | ✓                  |
+| Mobilenetv3 (timm)           | ✓               | ✓                  | ✓              | ✓                  | ✓               | ✓                      | ✓                  | ✓                  |
+| DeiT (huggingface)           | ✓               | ✓                  | ✓              | ✓                  | ✓               | ✓                      | ✓                  | ✓                  |
+| VanillaNet (timm)            | -               | -                  | ✓              | ✓                  | -               | ✓                      | ✓                  | ✓                  |
+| Swin (huggingface)           | ✓               | ✓                  | ✓              | ✓                  | ✓               | ✓                      | ✓                  | ✓                  |
+| YoloX (mmyolo/mmdet)         | -               | ✓                  | -              | ✓                  | ✓               | ✓                      | ✓                  | -                  |
+| RTMDet (mmyolo/mmdet)        | -               | ✓                  | -              | ✓                  | ✓               | ✓                      | -                  | -                  |
+| Yolov8 (mmyolo)              | -               | -                  | -              | ✓                  | -               | ✓                      | -                  | -                  |
+
+## LLM Compression
+
+| Model   	| AWQ 	| LMQuant (QoQ) 	| AQLM 	| TensorRT 	| Exllama 	| MLC-LLM 	| FLAP 	|
+|---------	|-----	|---------------	|------	|----------	|---------	|---------	|------	|
+| LLaMA   	| ✓   	| ✓             	| ✓    	| ✓        	| ✓       	| ✓       	| ✓    	|
+| LlaMA-2 	| ✓   	| ✓             	| ✓    	| ✓        	| ✓       	| ✓       	| ✓    	|
+| Llama-3 	| ✓   	| ✓             	| ✓    	| -        	| ✓       	| -       	| ✓    	|
+| Vicuna  	| ✓   	| ✓             	| ✓    	| ✓        	| ✓       	| ✓       	| ✓    	|
+| Mistral 	| ✓   	| ✓             	| ✓    	| ✓        	| ✓       	| ✓       	| ✓    	|
+| Mixtral 	| ✓   	| ✓             	| ✓    	| ✓        	| ✓       	| ✓       	| ✓    	|
+| Gemma   	| ✓   	| -             	| ✓    	| ✓        	| -       	| -       	| -    	|
+
+## Adapt
```

Comment:
I think we can make this "Adaption" instead

Reply:
done.